### PR TITLE
fixes #980

### LINKED ISSUE #980: Add current URL on the error message
## The problem
The error message on a test failure doesn't display the URL where we are getting the error. 

## Details

The following message is helpful. It would be great to add the URL as well.

```
com.codeborne.selenide.ex.ElementNotFound: Element not found {#dashboardButton}
Expected: visible or transparent: visible or have css value opacity=0
Screenshot: file:/C:/_work/app/build/reports/tests/1570048960084.0.png
Page source: file:/C:/_work/app/build/reports/tests/1570048960084.0.html
Timeout: 4 s.
Caused by: NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"#dashboardButton"}

	at com.codeborne.selenide.impl.WebElementSource.createElementNotFoundError(WebElementSource.java:37)
	at com.codeborne.selenide.impl.ElementFinder.createElementNotFoundError(ElementFinder.java:100)
	at com.codeborne.selenide.impl.WebElementSource.checkCondition(WebElementSource.java:65)

```
 
## Possible Solution:

We may need to update com.codeborne.selenide.ex.UIAssertionError#uiDetails() and  com.codeborne.selenide.ex.ErrorMessages to pull the current url from driver object:

`driver.getWebDriver().getCurrentUrl();`

### PR DESCRIPTION
**Pull request by eroshenkoam**
## Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue.

Posible fixes #980

Action list
- [x] first draft
- [ ] provide tests
- [ ] refactor meta adding

### CODE REVIEW

**Comment by eroshenkoam** [comment]
@asolntsev could you accept my concept

**Comment by BorisOsipov** [comment]
Be careful
`String url = driver.getWebDriver().getCurrentUrl();`
will throw `UnsupportedCommandException` in appium

File: src/main/java/com/codeborne/selenide/ex/ErrorMessages.java [hunk 1]
```diff
@@ -39,10 +41,15 @@ static String actualValue(Condition condition, Driver driver, WebElement element
   }
 
   public static String screenshot(Driver driver) {
-    return screenshot(driver.config(), ScreenShotLaboratory.getInstance().formatScreenShotPath(driver));
+    String url = driver.getWebDriver().getCurrentUrl();
+    String screenshotPath = ScreenShotLaboratory.getInstance().formatScreenShotPath(driver);
+    return screenshot(driver.config(), screenshotPath);
```
Comment (Mart-Bogdan) [inline src/main/java/com/codeborne/selenide/ex/ErrorMessages.java:46]: You forgot to add `url` as the second parameter, but the local variable exists.

**Review by Mart-Bogdan — CHANGES_REQUESTED** [review]
There is a single bug that makes this code doing nothing.

**Pull request closed without merging**